src/vivarium/examples/disease_model/population.py

```diff
@@ -142,6 +142,9 @@ def on_time_step(self, event: Event) -> None:
             representing the simulants affected by the event and timing
             information.
         """
-        population = self.population_view.get(event.index, query="alive == 'alive'")
+        # TODO: clarify whether we should require columns needed for query or if we
```

Comment:
We should sort this out as part of this PR! I think it would be pretty difficult to parse the query string since there are MANY ways you can build those up (i.e. we can't just separate on "AND" even though that's usually what we do)